Makefile

```diff
@@ -60,11 +60,11 @@ website: docs ## Render the website
 
 .PHONY: test
 test: ## Run unit tests
-# Given the nature of generative tests the test timeout was increased from 500ms
+	@go test -race -covermode=atomic -coverprofile=coverage-unit.out -timeout 500ms -tags=unit ./...
```

Comment:
What's the benefit of splitting up the coverage reports?

Reply:
So that the next `go test...` doesn't overwrite the coverage report from the previous `go test...`